subscribed_topics:
  # Topics in Rgbd-T synchronizer node
  input_topic: /kinect/depth_registered/points

  # Temporary topic for rosbags
  # input_thermal_topic: /flir_lepton/fused_msg
  # Real topic after integration
  input_thermal_topic: /rpi2/thermal/fused_msg

  # Topics in synchronizer node
  synchronized_topic: synchronized/message/info
  unlock_topic: unlock_rgb_depth_synchronizer
  subscribe_to_input: subscribe_to_input
  leave_subscription_to_input: leave_subscription_to_input

  # From thermal cropper
  thermal_unlock_synchronizer_topic: thermal/unlock/synchronizer

published_topics:
  # Topics in synchronizer node
  point_cloud_internal_topic: synchronized/depth/points
  depth_image_topic: synchronized/depth/image
  rgb_image_topic: synchronized/rgb/image
  thermal_info_topic: synchronized/thermal/image

  # To thermal cropper
  rgb_depth_images_cropper_topic: rgb/depth/images/cropper

  # Topics in Rgbd-T synchronizer node
  synchronized_topic: synchronized/message/info
